packages/db/docker-compose.yml

```diff
@@ -13,7 +13,12 @@ services:
     volumes:
       - 100xdevs:/data/postgres
     restart: unless-stopped
-
+  
+  redis:
```

Comment:
Can you remove redis portion

Reply:
there was redis url written already written in env.example.. I thought redis is also being used.... No worries let me remove it!